pkg/systemd/logs.jsx

```diff
@@ -55,36 +55,45 @@ import "./logs.scss";
 const _ = cockpit.gettext;
 
 const timeFilterOptions = [
-    { key: "boot", value: 0, toString: () => _("Current boot"), },
-    { key: "boot", value: "-1", toString: () => _("Previous boot") },
-    { key: "since", value: "-24hours", toString: () => _("Last 24 hours"), default: true },
-    { key: "since", value: "-7days", toString: () => _("Last 7 days") },
+    { value: "current", content: _("Current boot") },
+    { value: "previous", content: _("Previous boot") },
+    { value: "last24h", content: _("Last 24 hours") },
+    { value: "last7d", content: _("Last 7 days") },
 ];
 
+const timeFilterParams = {
+    current: { param: "boot", value: 0 },
```

Comment:
Could we somehow allow "any additional junk" in the type declaration of MenuItem, so that we can continue to add these extra fields right in their declaration? and they are passed through event handlers opaquely?

Reply:
What about allowing
```
{ 
  content: _("Current boot"),
  value: { key: "boot", value: 0 },
}
```

and we use `content` as the React key?

Reply:
FTR, I'm completely happy with the `value`s here. Restricting them to strings (or perhaps also numbers) keeps things easy and non-ambiguous. I was just asking if we could merge` timeFilterParams` into `timeFilterOptions` by just adding extra `param` and `systemd_value` (or similar) keys.

Both value and content are fine for the key, but I think `value` is a bit cleaner (i.e. the status quo)

Reply:
In other words, can we declare `SimpleSelectMenuOption` as content: string, value: string | number, plus "any additional stuff" like `[string]: unknown`? 

If that's difficult/impractical then ignore me. I was just wondering if object types can be partially specified and partially opaque.

Reply:
Done. `content` can be any React.ReactNode so it can't always be used as a key, so we fall back to the ... index. :-/ But a explicit key in SimpleSelectOption is also supported.

Reply:
> In other words, can we declare SimpleSelectMenuOption as content: string, value: string | number, plus "any additional stuff" like [string]: unknown?

Hmm, we could, but I think it's not a good idea. It already has all the options from SelectOption, which inherits from MenuItem, and I think the risk of collision is just too big. For example, both `key` and `value` are already taken.

The old code also didn't do this. It didn't have 
```
<SelectOption key='boot' value='-1'>Previous boot</SelectOption>
```
in it, but constructed it from timeFilterOptions.

Reply:
I see. I clearly don't understand this enough, and it looks fine - I was just curious.  So let's just keep this. Thanks!